web/src/locales/en.json

```
@@ -124,5 +124,13 @@
 	"employees.notFound.description": "The employee you are looking for does not exist.",
 	"employees.delete.success": "Employee deleted successfully",
 	"employees.delete.conflict.title": "Conflict",
-	"employees.delete.conflict.description": "This employee cannot be deleted because it's associated with a report or a route."
+	"employees.delete.conflict.description": "This employee cannot be deleted because it's associated with a report or a route.",
```

This was updated, now the report can not generate any conflicts:
```suggestion
	"employees.delete.conflict.description": "This employee cannot be deleted because it's associated with a route.",
```